core/object/object.cpp

```diff
@@ -1347,7 +1347,7 @@ Error Object::connect(const StringName &p_signal, const Callable &p_callable, ui
 		s = &signal_map[p_signal];
 	}
 
-	Callable target = p_callable;
+	const Callable &target = p_callable;
```

Comment:
This should be replaced in-place, as it is the direct argument

Reply:
Handled in #85071.